src/pdf_beamtime/config/joint_poses.yaml

```diff
@@ -1,3 +1,4 @@
 pdf_beamtime_server:
   ros__parameters:
     home_angles: [-0.11536, -1.783732, 0.38816, -1.75492, 0.11484, 3.14159] #For real robot
+    gripper_present: false
```

Comment:
Make sure that this parameter can be set at launch time. 
(It probably can, but please confirm).

Reply:
Parameters gripper_present can be changed via the command
`ros2 param set gripper_present true`